grails-app/domain/org/broadinstitute/orsp/Issue.groovy

```diff
@@ -142,6 +142,8 @@ class Issue implements LogicalDelete<Issue> {
 
     transient Collection<String> getPMs() { getExtraProperties().findAll { it.name == IssueExtraProperty.PM }.collect { it.value } }
 
+    transient Collection<String> getCollaborators() { getExtraProperties().findAll { it.name == "collaborator" }.collect { it.value } }
```

Comment:
Nit: We have a string property for this: `IssueExtraProperty.COLLABORATOR`
Also, we have some custom collaborator logic in `Issue.getExtraPropertiesMap()` that I'm having trouble following. @LeoForconesi - Can you elaborate here on if we're missing anything? I think we could use some comments around why `getExtraPropertiesMap()` is doing what it is doing if we have a `Issue.getCollaborators()` method.

Reply:
@rushtong @dags  
`getExtraPropertiesMap()`'s goal is to obtain some sort of a "maped" object {key: value} style to be send directly to the Ui, and save some parse methods in this end. 
I belive that if we now have a `getCollaborators` method, maybe we could use it thogether with `getExtraPropertiesMap` ?. 
Our Ui expects a "Collaborators" extraProperty which contains an array of collaborators string names. Here's a [link](https://github.com/broadinstitute/orsp-pub/blob/feature/src/main/webapp/projectReview/ProjectReview.js#L129) to see how it's used on the front end. 
Does it clarify your doubts? Let me know

Reply:
Thanks @LeoForconesi. I spent a little more time looking at that and I'll submit a follow-up PR that I think will clean it up. The essence of that method is to copy all `collaborator` properties into a `collaborators` map entry.